File: pinyougou/pinyougou-manage-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {

    //查询单个信息
    this.findOne = function (id) {
        return $http.get('../brand/findOne.do?id='+id);
    }

    // 分页查询
    this.findPage = function (page, rows) {
        return $http.get("../brand/findPage.do?page=" + page + "&rows=" +rows);
    }

    //搜索
    this.search = function (page,rows,searchEntity) {
        return $http.post('../brand/search.do?page='+page+'&rows='+rows,searchEntity);
    }

    //添加
    this.add = function (entity) {
        return $http.post('../brand/add.do',entity)
    }

    //修改
    this.update = function (entity) {
        return $http.post('../brand/update.do',entity)
    }

    //批量删除
    this.delete = function (selectedIds) {
        return $http.get('../brand/delete.do?ids='+selectedIds);
    }
});

File: pinyougou/pinyougou-sellergoods/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import com.pinyougou.service.BaseService;
import com.pinyougou.vo.PageResult;

public interface BrandService extends BaseService<TbBrand>{

    /**
     * 根据条件查询
     * @param brand
     * @param page
     * @param rows
     * @return
     */
    public PageResult search(TbBrand brand,Integer page,Integer rows);

}
